portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyTestFile.testcase

```diff
@@ -0,0 +1,30 @@
+@component-name = "portal-web-forms-and-data-lists"
+definition {
+
+	property testray.main.component.name = "Forms";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	@description = "Set Screen size to custom and validate values"
+	@priority = 5
+	test CanDefineDXPCustomScreenSize {
+		MyTestFile.SimulationButton();
+
+		MyTestFile.CustonButton();
+
+		MyTestFile.EditWidth();
+
+		MyTestFile.EditHight();
+
+		MyTestFile.ApllyCustonButton();
```

Comment:
You can have all these macros together in one called `SetCustomScreenSize`, please update it